docs/getting-started-guides/mesos.md

```diff
@@ -0,0 +1,303 @@
+## Getting started with Kubernetes on Mesos
+
+<!-- TODO: Update, clean up. -->
+
+Mesos allows dynamic sharing of cluster resources between Kubernetes and other first-class Mesos frameworks such as [Hadoop][1], [Spark][2], and [Chronos][3].
+Mesos ensures applications running on your cluster are isolated and that resources are allocated fairly.
+
+Running Kubernetes on Mesos allows you to easily move Kubernetes workloads from one cloud provider to another to your own physical datacenter.
+
+This tutorial will walk you through setting up Kubernetes on a Mesos cluster on [Google Cloud Plaform][4].
+It provides a step by step walk through of adding Kubernetes to a Mesos cluster and running the classic GuestBook demo application.
+The walkthrough presented here is based on the v0.4.x series of the Kubernetes-Mesos project, which itself is based on Kubernetes v0.11.0.
+
+### Prerequisites
+
+* Understanding of [Apache Mesos][11]
+* Mesos cluster on [Google Compute Engine][5]
+* Identify the Mesos master node external IP from Mesosphere [cluster launch pad][12]
+* A [VPN connection to the cluster][6], or setup firewall rules for direct access (used in this doc).
+
+### Setup Access to Mesos Admin GUI
+This doc will refer to the running Mesos cluster repeatedly.  It's a good idea to make the Mesos admin GUI accessible via the browser.  First, take note of the network name attached to your Mesos cluster (either via the Google Cloud GUI) or with `gcloud` tool as follows:
+
+```bash
+gcloud compute firewall-rules list
+```
+Next, add a firewall-rule to open traffic on port 5050.  This will allow you to access Mesos master GUI via the browser.
+
+```bash
+gcloud compute firewall-rules create mesos-master-gui --network ${network_name}  --source-ranges 0.0.0.0/0 --allow tcp:5050
+```
+Once the rule is applied, point your browser to your Mesos master's IP address (see pre-requisites).  You should see the Mesos GUI displayed.
```

Comment:
Oops, two comments on my above comment:
(1) where I said http://:5050 I meant `http://<master external IP>:5050`

(2) the email I received when I set up the cluster only made reference to VPN ("Next: set up VPN to access consoles"); if there are actually two paths forward on GCE (VPN or firewall rule) then maybe the email should mention both.